src/redis_cmd.cc

```diff
@@ -1517,16 +1522,16 @@ class CommandPop : public Commander {
     if (args.size() == 2) {
       return Status::OK();
     }
-    try {
-      int32_t v = std::stol(args[2]);
-      if (v < 0) {
-        return Status(Status::RedisParseErr, errValueMustBePositive);
-      }
-      count_ = v;
-      with_count_ = true;
-    } catch (const std::exception& ) {
+    auto parseResult = ParseInt<int32_t>(args[2], 10);
+    if (!parseResult) {
       return Status(Status::RedisParseErr, errValueNotInterger);
     }
+    int32_t v = *parseResult;
```

Comment:
actually we prefer underscore-case for local variable, and for here, you can use `v` instead of `parseResult`